run.py

```diff
@@ -77,7 +77,10 @@ def create_tables():
         MentorshipRelationExtensionModel,
     )
 
-    db.create_all()
+    db.session.execute("CREATE SCHEMA IF NOT EXISTS bit_schema")
```

Comment:
Cool 🎉. I learn something here (creating db from inside python 👍👍).
However, these changes are assuming that db is created within docker, right? 
So, I don't think these will work for those who wants to run the application locally, right?

Reply:
> Cool 🎉. I learn something here (creating db from inside python 👍👍).
> However, these changes are assuming that db is created within docker, right?
> So, I don't think these will work for those who wants to run the application locally, 🤔.

Nope nope this runs after the connection to the database has been validated so as long as the database uri is setup correct this should work for all users : ).